File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Xóa số/HH_XOASO.cs
using System;
namespace HH_XOASO
{
    public class Program
    {
        static void Main(string[] args)
        {
            string N = Console.ReadLine();
            string maxNumber = "";
    
            for (int i = 0; i < N.Length; i++)
            {
                string temp = N.Substring(0, i) + N.Substring(i + 1);
    
                if (maxNumber == "" || String.Compare(temp, maxNumber) > 0)
                {
                    maxNumber = temp;
                }
            }
            
            Console.Write(N.Length == 1 ? N : maxNumber);
        }
    }

}
    

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Xóa số/HH_XOASO.py
N = input()
max_number = ""

for i in range(len(N)):
    temp = N[:i] + N[i+1:]

    if max_number == "" or temp > max_number:
        max_number = temp

print(N if len(N) == 1 else max_number, end= "")
